src/components/UserProfile/AddNewUserProfile/UserProfileAdd.jsx

```diff
@@ -430,14 +436,25 @@ class AddUserProfile extends Component {
     if (googleDoc) {
       userData.adminLinks.push({ Name: 'Google Doc', Link: googleDoc })
     }
-
-    createUser(userData)
+    if(this.state.userProfile.phoneNumber !== null)
```

Comment:
can we move all these conditions to a function and use that function here before calling createUser?

Reply:
conditions moved to:  inputsHaveLength()